### PR TITLE
Bugfix/1864 datepicker display

### LINKED ISSUE #1864: Datepicker: design issue when no label in IE11 and Edge
Unfortunately IE11 and Edge still exist, and the Datepicker renders extra space when there's no `label` attr provided, which breaks our design...

See screenshot from offic Storybook.

Possible workaround:

```css
axa-datepicker article.m-datepicker {
  display: block; // override "display: inline" which causes problem in IE11+Edge
}
```

Question:

why does this css declaration exists in Datepicker's scss file? What is its purpose?
image

**Screenshot:**

image

### PR DESCRIPTION
**Pull request by danyball**
Fixes #1864

# Done is when (DoD):
- [ ] Modifications within components are reflected by tests.
- [x] A self-review of the code has been performed.
- [ ] The modified component properties have been added to the typescript typings.
- [ ] Potential design changes have been reviewed by a designer.
- [x] The modifications are well documented (README.md, etc.) and showcased in the stories.
- [x] The modifications are documented in the code, particularly in hard-to-understand areas (focus on **WHY**).
- [x] The modifications are shortly added to CHANGELOG.md with the issue number. AFTER MERGE because of conflict to other PR
- [x] The modifications still work in IE.
- [x] The modifications are according to our [Best Practices](https://github.com/axa-ch/patterns-library/blob/develop/CONTRIBUTION.md#best-practices)

### CODE REVIEW

File: src/components/20-molecules/datepicker/index.scss [hunk 1]
```diff
@@ -80,7 +80,7 @@ axa-datepicker {
 
 .m-datepicker {
   position: relative;
-  display: inline;
+  display: block;
```
Comment (LucaMele) [inline src/components/20-molecules/datepicker/index.scss:83]: This needs to be tested in a form

Comment (danyball) [reply src/components/20-molecules/datepicker/index.scss:83]: Looks quite nice: https://axa-ch.github.io/plib-feature/bugfix/1864_datepicker_display/?path=/story/components-datepicker-demos--in-a-form

Comment (danyball) [reply src/components/20-molecules/datepicker/index.scss:83]: Or what to check in a form?

Comment (LucaMele) [reply src/components/20-molecules/datepicker/index.scss:83]: Aem forms. Just to make sure is ok there

Comment (danyball) [reply src/components/20-molecules/datepicker/index.scss:83]: Have changed it at chrome-dev-tools at acc stage at a aem form. Do not change anything there. Is that test ok?

Comment (danyball) [reply src/components/20-molecules/datepicker/index.scss:83]: At IE on aem forms is looks better because extra space on top of datepicker is gone.

**Review by 01231 — APPROVED** [review]
LGTM if Luca is happy how you tested it

**Pull request merged**